src/pages/EnrollmentList.js

```diff
@@ -23,7 +23,7 @@ export const TARGET_API_LABELS = {
   franceconnect: 'FranceConnect',
   api_droits_cnam: 'API Droits CNAM',
   dgfip: 'API Impôt particulier',
-  api_impot_particulier: 'Impôt particulier 1/2',
+  api_impot_particulier: 'Impôt particulier',
   api_impot_particulier_step2: 'Impôt particulier 2/2',
```

Comment:
`Impôt particulier 2/2` --> `Impôt particulier (suite)` ?
**or**
`Impôt particulier pré-validation` ?

Reply:
Seen IRL, no solution has been found for now.